Solutions/CohesitySecurity/Tests/helios.py

```diff
@@ -1,12 +1,47 @@
 #!/usr/bin/env python3
 
 import requests
+import datetime
+import datetime
+import time
+
+
+def get_alerts_details(alert_ids, apiKey):
```

Comment:
We should not be needing to pass around apiKey for every function. Ideally, we should have the functions in this file clubbed into a class which accepts the API key in the constructor and that should be re-used. I'll leave it upto you if you want to do it now or later.